### PR TITLE
Check if input is missing, add name and examples for commander help

### PR DESCRIPTION
**Pull request by joelanman**
To check for missing input I had to do quite a big refactor. Commander only runs `action` if the required input exists, otherwise it just silently exits.

I'm assigning the input to `program.input` in the `action`. Then I check if that exists at all in the main script.

While doing this work I noticed that Commander gets the name of the command in `help` from the filename, so I've renamed it to `wshot.js`.

I've also added a `description` with examples.

I have to say, Commander's documentation is hard to follow, and implementing this check for `input` seems harder than it should be. [Yargs](https://github.com/yargs/yargs) may be a better alternative.

### CODE REVIEW

**Comment by joelanman** [comment]
Is that possible? `dist/wshot.js` is what gets run in the end. If it was missing from the repo, `npm install -g wshot` wouldn't include it would it?

**Comment by quis** [comment]
I reckon something in here would make it possible: https://docs.npmjs.com/misc/scripts
